src/doom/d_compat.c

```diff
@@ -268,19 +268,19 @@ void ChangeTrigTablesToDoom1_0()
     coarsecosine = &coarsesine1_0[64];
 }
 
-extern fixed_t forwardmove[2], sidemove[2];
-
-// Doom 1.0 speed values, scaled by 1/900
-fixed_t         forwardmove10[2] = {51200/900, 102400/900}; 
-fixed_t         sidemove10[2] = {49152/900, 81920/900}; 
-
 fixed_t turbo_scale = 100;
 fixed_t cmd_move_scale = 2048;
 
-int sfx_getpow_1_2 = sfx_getpow;
+int compat_sfx_getpow = sfx_getpow;
```

Comment:
I suppose I could have done this by renaming `sfx_getpow` in `sfxenum_t` and naming this `sfx_getpow` directly, but I don't really want to mess with the enum in that way.